client/src/rpc_filter.rs

```diff
@@ -16,21 +16,28 @@ impl RpcFilterType {
                 match encoding {
                     MemcmpEncoding::Binary => {
                         let MemcmpEncodedBytes::Binary(bytes) = &compare.bytes;
-                        bs58::decode(&bytes)
-                            .into_vec()
-                            .map(|_| ())
-                            .map_err(|e| e.into())
+
+                        if bytes.len() > 128 {
```

Comment:
176 would allow for a 128 byte decoded string

Reply:
I picked 128 since we use that same limit for getAccountInfo/getMultipleAccounts/elsewhere
Don't super strongly about 176, since one can provide multiple filters.

Err, I think that's another thing to fix here as well!  We don't limit the max number of filters yet

Reply:
Filters now limited with the latest commit here.  8 seems like a lot.

Reply:
8 does seem like a lot.

Reply:
Hmm, yeah I'll drop this to 4 for now.  4 also seems like a lot